### ISSUE TITLE
如何设置单卡训练

### ISSUE THREAD

**Issue by zhou-wj** [PaddlePaddle/Paddle#18281]
- 版本、环境信息：
    1）PaddlePaddle版本：1.4.1
    2）GPU：CUDA Capability: 61, Driver API Version: 10.1, Runtime API Version: 9.0； cuDNN Version: 7.0.
    3）系统环境：Ubuntu 16.04.6
    4）Python版本号：3.5.1

训练脚本对https://github.com/PaddlePaddle/LARK/blob/develop/ERNIE/script/run_lcqmc.sh略做改动：

> set -eux
#export FLAGS_sync_nccl_allreduce=1
export CUDA_VISIBLE_DEVICES=0
export NV_GPU=0
MODEL_PATH=/home/zhouwenjie/Paddle/ERNIE_stable-1.0.1
TASK_DATA_PATH=/home/zhouwenjie/Paddle/data/testData
python -u run_classifier.py ...

查看gpu使用情况时确实只使用了卡0，但是报错如下：

> -----------  Configuration Arguments -----------
batch_size: 32
checkpoints: ./checkpoints
dev_set: /home/zhouwenjie/Paddle/data/testData/dev.tsv
do_lower_case: True
do_test: True
do_train: True
do_val: True
epoch: 3
ernie_config_path: config/ernie_config.json
in_tokens: False
init_checkpoint: None
init_pretraining_params: /home/zhouwenjie/Paddle/ERNIE_stable-1.0.1/params
label_map_config: None
learning_rate: 2e-05
loss_scaling: 1.0
lr_scheduler: linear_warmup_decay
max_seq_len: 128
metrics: True
num_iteration_per_drop_scope: 1
num_labels: 2
random_seed: 1
save_steps: 1000
skip_steps: 10
test_set: /home/zhouwenjie/Paddle/data/testData/test.tsv
train_set: /home/zhouwenjie/Paddle/data/testData/train.tsv
use_cuda: True
use_fast_executor: False
use_fp16: False
validation_steps: 100
verbose: True
vocab_path: config/vocab.txt
warmup_proportion: 0.0
weight_decay: 0.0
attention_probs_dropout_prob: 0.1
hidden_act: relu
hidden_dropout_prob: 0.1
hidden_size: 768
initializer_range: 0.02
max_position_embeddings: 513
num_attention_heads: 12
num_hidden_layers: 12
type_vocab_size: 2
vocab_size: 18000
Device count: 1
Num train examples: 3
Max train steps: 0
Num warmup steps: 0
memory_optimize is deprecated. Use CompiledProgram and Executor
Theoretical memory usage in training: 5410.435 - 5668.075 MB
W0624 07:35:13.556089 100014 device_context.cc:261] Please NOTE: device: 0, CUDA Capability: 61, Driver API Version: 10.1, Runtime API Version: 9.0
W0624 07:35:13.559146 100014 device_context.cc:269] device: 0, cuDNN Version: 7.0.
Load pretraining parameters from /home/zhouwenjie/Paddle/ERNIE_stable-1.0.1/params.
ParallelExecutor is deprecated. Please use CompiledProgram and Executor. CompiledProgram is a central place for optimization and Executor is the unified executor. Example can be found in compiler.py.
W0624 07:35:13.889760 100014 graph.h:204] WARN: After a series of passes, the current graph can be quite different from OriginProgram. So, please avoid using the `OriginProgram()` method!
W0624 07:35:13.930651 100014 dynamic_loader.cc:107] Can not find library: libnccl.so. Please try to add the lib path to LD_LIBRARY_PATH.

查看之前的issue定位问题为缺少libnccl.so，但是之前的issue中有提到使用单卡训练时不需要libnccl.so，请问如何设置单卡训练？

**Comment by JiabinYang** [comment]
您可以使用最新Develop分支代码编译出来的Paddle，或者下载我们为您编译的最新开发分支whl，http://paddlepaddle.org/documentation/docs/zh/1.4/beginners_guide/install/Tables.html，或者敬请期待1.5版本发布，我们将在新的版本中修复这个问题，又或者您可以使用Executor而不使用CompiledProgram和Parallel Executor

**Comment by zhou-wj** [comment]
谢谢，我明白了
从源码看ERNIE似乎没有支持单卡模式，源码中使用了Parallel Executor
